docs/nginx-ingress-controller.md

```diff
@@ -61,6 +62,7 @@ spec:
    reportIngressStatus:
      enable: true
      externalService: my-nginx-ingress
+     ingressLink: my-ingresslink
```

Comment:
since `ingressLink` is mutually exclusive with `externalService`, better to remove it